Breeze.BreezeServer/BreezeRegistration.cs

```diff
@@ -202,5 +202,33 @@ public Transaction CreateBreezeRegistrationTx(Network network, byte[] data, Mone
 
             return sendTx;
         }
+
+        public bool VerifyCollateral(BreezeConfiguration config, out Money missingFunds)

```

Comment:
This is a good idea, but bear in mind a light wallet client's perspective in this scenario:

- Server has 250k coins in wallet address A
- Server performs registration transaction R, specifying the collateral address as A
- Light client C only now adds A to its watch-only wallet. The balance will appear as zero.
- Server will regard everything as ok, because its wallet balance is 250k from its perspective.
- No light clients will regard the registration as valid, unless the server sends the collateral back to itself.

Reply:
The intention is to give warning to the masternode operator when
a) they are running for the first time
b) they accidentally removed funds from the collateral address